forms-web-app/src/middleware/fetch-existing-appeal.js

```diff
@@ -17,6 +17,10 @@ module.exports = async (req, res, next) => {
   try {
     req.log.debug({ uuid: req.session.appeal.uuid }, 'Get existing appeal');
     req.session.appeal = await getExistingAppeal(req.session.appeal.uuid);
+
+    req.session.appeal['appeal-upload'] =
+      (req.session.appeal['appeal-upload'] && JSON.parse(req.session.appeal['appeal-upload'])) ||
+      {};
```

Comment:
This bit I really do not like. 

We will need one of these per upload field, and the one for multiple uploads is going to be a nasty map as best I can foresee. 

About the only saving grace here is that this is happening once, in this middleware, rather than per controller.